app/storybook/.storybook/preview.tsx

```diff
@@ -16,10 +18,62 @@ const GlobalStyles = () => (
         /* @TODO: find a way to customize it without !important dominance */
         color: #2e90fa !important;
       }
+      a.showCodeLink {
+        display: block;
+        margin-top: 20px;
+        cursor: pointer;
+        font-size: 0.9em;
+        color: #2e90fa;
+      }
     `}
   />
 )
 
+const withSource = (StoryFn, context) => {
```

Comment:
Initially, I tried to use [Canvas](https://storybook.js.org/docs/react/api/doc-block-canvas) and [Source](https://storybook.js.org/docs/react/api/doc-block-source) blocks but it didn't work due context issues:

<img width="980" alt="image" src="https://github.com/propeldata/ui-kit/assets/55801864/643823c2-9e81-4d41-b89f-5ce9a01628ec">

I found some promising solutions to provide the proper context:
1. Based on [loaders](https://github.com/storybookjs/storybook/blob/a3cdabb025524822807318bc137f69be006596c2/code/ui/.storybook/preview.tsx#L98)
2. Based on [external documentation](https://github.com/storybookjs/storybook/discussions/24057)) 

I got some intermediate results with the loader but the output was [quite the same](https://github.com/storybookjs/storybook/issues/22922):

<img width="450" alt="image" src="https://github.com/propeldata/ui-kit/assets/55801864/e47f1c6b-2934-41e4-b948-a38c287f0393">

So, I dug  deeper into the Story's context object and found the following:

### Story 1
<img width="852" alt="image" src="https://github.com/propeldata/ui-kit/assets/55801864/b5f0e873-f9a8-4240-b8d6-9a3f12582196">

### Story 2
<img width="785" alt="image" src="https://github.com/propeldata/ui-kit/assets/55801864/785426aa-3763-4f70-8716-39038bfe1261">

`context.parameters.docs.source.originalSource` prop of the Story context has a source code. Unfortunately, not in a perfect form:
1. `originalSource` is not a valid JSON
2. An original component inside has not mapped `args`

The solution:
1. Parse a story source from `originalSource`
2. Replace `{...args}` with the actual params imported directly from Story's module
3. Format it via standalone prettier
4. Pass the output as a string prop to `Source` doc component.
5. Add "Show source" button to show hidden by default source code.